packages/button/src/styles.tsx

```diff
@@ -133,3 +135,37 @@ export const getButtonStyle = (
       }
   }
 }
+
+export function getIconAttachmentStyle(
+  height: number,
+  iconPosition?: ButtonIconPosition
+) {
+  return {
+    display: 'flex',
```

Comment:
1) `flex` should not be added automatically to all icon-enabled buttons
2) we should retain former developer experience, i.e. not adding `full` should make button inlined